payments-core/src/main/java/com/stripe/android/payments/core/analytics/RealErrorReporter.kt

```diff
@@ -18,6 +18,7 @@ class RealErrorReporter @Inject constructor(
         val additionalParams = mapOf(
             "analyticsValue" to stripeException.analyticsValue(),
             "statusCode" to stripeException.statusCode.toString(),
+            "requestId" to stripeException.requestId,
```

Comment:
We probably want to add this to the other places we report errors as well.

Reply:
Good idea. Unfortunately, since analytics from different parts of the sdk are all sent differently, I think we'd need to add it to each place individually

Created a jira to track though: https://jira.corp.stripe.com/browse/MOBILESDK-1919